### PR TITLE
add address display to form answers page

### PR DESCRIPTION
**Pull request by anabebs**
**What**  
small PR to update the person widget so that it uses both address and addresses when required.
This will fix the problem of the address not displaying on the form answers page.

### CODE REVIEW

File: components/PersonWidget/PersonWidget.tsx [hunk 1]
```diff
@@ -18,10 +18,12 @@ interface Props {
...
 const PersonWidget = ({ person }: Props): React.ReactElement => {
   const dateOfBirth = prettyDate(person?.dateOfBirth ?? '');
-
-  const displayAddress = person?.addresses?.[0];
+  const displayAddress = person?.address?.address;
+  const displayPostcode = person?.address?.postcode;
+  const displayAddresses = person?.addresses?.[0];
 
   if (person) {
+    console.log(displayPostcode);
```
Comment (jhackett1) [inline components/PersonWidget/PersonWidget.tsx:26]: errant console.log

File: components/PersonWidget/PersonWidget.tsx [hunk 2]
```diff
@@ -30,14 +32,11 @@ const PersonWidget = ({ person }: Props): React.ReactElement => {
...
-
-        {displayAddress && (
-          <p className={`lbh-body-s ${s.paragraph}`}>
-            {displayAddress.addressLines}
-            <br />
-            {displayAddress.postCode}
-          </p>
-        )}
+        <p className={`lbh-body-s ${s.paragraph}`}>
+          {displayAddress || displayAddresses?.addressLines}
```
Comment (jhackett1) [inline components/PersonWidget/PersonWidget.tsx:36]: is it ever possible to have no address on a person at all?

**Review by jhackett1 — APPROVED** [review]
looks fine apart from minor gripes

File: components/PersonWidget/PersonWidget.tsx (hunk 2 above)
Comment (anabebs) [reply components/PersonWidget/PersonWidget.tsx:36]: maybe but i thought all the question marks in the consts would deal with that? Do they not?

Comment (jhackett1) [reply components/PersonWidget/PersonWidget.tsx:36]: you'd still end up with an empty div if there is no address to show - if it's not possible for a person to have no address (if it's always a required field) then that's fine

**Pull request merged**